tests/integration/test_settings_profile/test.py

```diff
@@ -454,22 +454,41 @@ def test_show_profiles():
     assert instance.query("SHOW PROFILES") == "default\nreadonly\nxyz\n"
 
     assert instance.query("SHOW CREATE PROFILE xyz") == "CREATE SETTINGS PROFILE xyz\n"
+
+    query_possible_response = [
+        "CREATE SETTINGS PROFILE default\n",
+        "CREATE SETTINGS PROFILE default SETTINGS allow_experimental_analyzer = true\n",
+    ]
     assert (
         instance.query("SHOW CREATE SETTINGS PROFILE default")
-        == "CREATE SETTINGS PROFILE default\n"
+        in query_possible_response
     )
-    assert (
-        instance.query("SHOW CREATE PROFILES") == "CREATE SETTINGS PROFILE default\n"
+
+    query_possible_response = [
+        "CREATE SETTINGS PROFILE default\n"
         "CREATE SETTINGS PROFILE readonly SETTINGS readonly = 1\n"
-        "CREATE SETTINGS PROFILE xyz\n"
-    )
+        "CREATE SETTINGS PROFILE xyz\n",
+        "CREATE SETTINGS PROFILE default SETTINGS allow_experimental_analyzer = true\n"
+        "CREATE SETTINGS PROFILE readonly SETTINGS readonly = 1\n"
+        "CREATE SETTINGS PROFILE xyz\n",
```

Comment:
are these two needed?

Reply:
Yes, they are needed. As you can see array contains 2 strings, but they are split into several literals for readability. @alexey-milovidov I'll revert your commit because the test will fail with it.